Makefile

```diff
@@ -54,7 +54,7 @@ ifeq ($(DB),$(SQLITE_PROFILE))
 endif
 
 define run_docker_compose_command
-	$(DOCKER_COMPOSE_ENV_VARS) docker compose -f $(DOCKER_COMPOSE_FILE) $(1)
+	$(DOCKER_COMPOSE_ENV_VARS) docker-compose -f $(DOCKER_COMPOSE_FILE) $(1)
```

Comment:
https://docs.docker.com/compose/compose-v2/#differences-between-compose-v1-and-compose-v2

Reply:
Thanks for pointing that out! I've reverted that change.